raspberry.rb

```diff
@@ -0,0 +1,124 @@
+class Raspberry
+  @@states = %w[absence bloom green red]
+
+  def initialize(index)
+    @index = index
+    @state = @@states.first
+  end
+
+  def grow!
+    i = @@states.index(@state)
+    if i == 3
+      puts "Малина уже созрела!"
+    else
+      @state = @@states[i + 1]
+    end
+  end
+
+  def ripe?
+    @state.eql?(@@states[3])
+  end
+end
+
+class RaspberryBush
+
+  def initialize(raspberries_amount)
+    @raspberry_bash = Array.new(raspberries_amount)
+    (0..raspberries_amount).each { |i|
+      @raspberry_bash[i] = Raspberry.new(i)
+    }
```

Comment:
Why you initialize empty array first?

Reply:
And don't use curly braces if the body doesn't fit in one line.

Reply:
Instead `(0..raspberries_amount).each` it is better to code it like `raspberries_amount.times`